src/naming/dune

```diff
@@ -11,6 +11,8 @@
    ast_generic
    semgrep.core ; for Lang
    parser_javascript.ast ; ugly for Ast_js.default_entity
+   semgrep.il
+   pfff-lang_GENERIC-analyze
```

Comment:
(Please excuse my lack of knowledge) What's the purpose of this `pfff-lang_GENERIC-analyze` dependency?

Reply:
Ooh! Thanks for catching that. It was needed because `naming` was in a different module than `analyze`. But I moved the implementation to `analyze`, so this change in `naming` isn't needed anymore.